### PR TITLE
Add a test-mocha target that uses mocha-dominate for JSDom support.

### PR DESCRIPTION
**Pull request by alexjeffburke**

### CODE REVIEW

**Comment by sunesimonsen** [comment]
Will imocha work with this?

**Comment by alexjeffburke** [comment]
@sunesimonsen oh that’d be a nice re-arragement! Re imocha, not sure but that is an extremely good Q - let me look into that as part of confidence building and if it checks out I’ll look at the test matrix bit too :)

**Comment by sunesimonsen** [comment]
👌

File: package.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     "build-tests": "babel --out-dir build --quiet test/index.spec.js",
     "lint": "eslint .",
     "prepare": "npm run build",
-    "test": "jest && npm run test:documentation",
+    "test": "npm run test-jest && npm run test-mocha && npm run test:documentation",
+    "test-jest": "jest",
+    "test-mocha": "mocha -r mocha-dominate -r babel-register --opts ./test/mocha.empty.opts",
```
Comment (sunesimonsen) [inline package.json]: Why is it necessary to point to empty opts?

Comment (sunesimonsen) [reply package.json]: I can make it work by adding the config to the existing opts file.

Comment (sunesimonsen) [reply package.json]: Screenshot 2019-06-26 at 20 42 21

Comment (sunesimonsen) [reply package.json]: It is still super fast.

**Comment by sunesimonsen** [comment]
@alexjeffburke I added a commit on top, feel free to remove it again if you disagrees.

**Pull request merged**